### ISSUE TITLE
Failed to export labels package in SLEAP 1.6.5 GUI

### ISSUE THREAD

**Issue by rufeili** [talmolab/sleap#2854]
Hi there,

Thank you for maintaining SLEAP! I posted this originally in [Discussions](https://github.com/talmolab/sleap/discussions/2851) but realized it's probably a bug report, so I'm moving it here.
I recently upgraded to SLEAP 1.6.5 and when I exported training job package in the GUI for remote training, an error showed up "Failed to export labels package: ExportPackageThread.run..on_progress() takes 2 positional arguments but 3 were given". In comparison, export worked fine on the same .slp file when using SLEAP 1.6.1 GUI (sleap-io 0.6.4) in the past.

It seems that sleap-io 0.9.2 changed progress_callback to (current, total, phase), but SLEAP 1.6.5's ExportPackageThread.run still defines (current, total). I wonder if this could be a bug and could be fixed soon? 

System
OS: Windows 11
SLEAP 1.6.5 · sleap-nn 0.3.3 · sleap-io 0.9.2
Python 3.13.11 · PyTorch 2.13.0+cu132 · CUDA 13.2 · RTX 4090
Install: uv tool install --python 3.13 "sleap[nn]==1.6.5" --torch-backend auto

**Comment by alicup29** [comment]
Hello @rufeili !

Thanks for the catch. As you mentioned, `sleap-io` 0.9.x changed the `progress_callback` signature used during SLP export/embedding from `(current, total)` to `(current, total, phase)`. We've updated the callback in this [sleap PR](https://github.com/talmolab/sleap/pull/2858), and it should be working fine from the `develop` branch!

```bash
uv tool install --force --python 3.13 "sleap[nn] @ git+https://github.com/talmolab/sleap.git@develop" --torch-backend auto
```
(`--force` reinstalls over your existing 1.6.5.) 

After that, Export Training Job Package and the other embed-frame package exports should complete normally, with no `TypeError`. Let us know if that works for you!

Thanks,
Amick

**Comment by rufeili** [comment]
Hi Amick, 

This works for me! Thank you!

All the best,
Rufei
